### ISSUE TITLE
can't create video

### ISSUE THREAD

**Issue by michael7908** [alembics/disco-diffusion#74]

**Comment by michael7908** [comment]
Thanks for creating such a powerful AI tool and I came across the issue that even I uncheck the skip_video_for_run_all box and the Disco Diffusion v5.2 [w/ VR Mode still can't create a video. 
Could you please figure out the reason? Many thanks.

**Comment by yizhangliu** [comment]
Maybe  you should check your  ffmpeg carefully.

**Comment by minakshimathpal** [comment]
@michael7908 
Hello there. I am new to disco diffusion. I am trying to create a video and I have set animation mode as 3D. My colab pro notebook is running for last 6hrs (GPU with High ram, Tesla t4) but still just 4%(180/5000) complete. Is this normal.

**Comment by aletts** [comment]
@minakshimathpal Yes, that sounds pretty normal. It's common for each image to take around 2 minutes to generate, and that's what you're seeing so far.
For an overall approach to yield quicker generation speed, you could try enabling turbo mode.. and also plan to use a frame interpolation solution such as FlowFrames (free software for Windows) or Google FILM frame-interpolation. This way, you can get smooth video without needing as many frames generated.
